cargo-miri/bin.rs

```diff
@@ -575,18 +577,29 @@ fn phase_cargo_miri(mut args: env::Args) {
     // so we cannot detect subcommands later.
     let subcommand = match args.next().as_deref() {
         Some("test" | "t") => MiriCommand::Test,
+        Some("nextest") => {
+            // The next argument is the nextest subcommand.
```

Comment:
Why is this needed? We anyway forward all the arguments to the actual cargo command. So what goes wrong if we treat `nextest` like we treat `test`?

Reply:
Basically I was hoping that, after https://github.com/rust-lang/miri/pull/2399, all we would have to do is add `nextest` to the list of forwarded commands, and be done. Does that work?

Reply:
Almost -- the only restriction is that `nextest` requires `list`/`run` to come before `--target`, not after it. So we have to grab the next argument from the list.

Just pushed an update which hopefully addresses this in a clean manner.